src/components/CartTotals.js

```diff
@@ -7,6 +7,7 @@ import { Link } from 'react-router-dom';
 
 const CartTotals = () => {
   const { total_amount, shipping_fee } = useCartContext();
+  // TODO: should tax maybe be a percentage of the subtotal? I suppose this may be different in different regions of the world.
```

Comment:
Also, since you need this same value in the `Orders` page, this might be an example of state that you want to "lift up" the React tree. One way would be to have it live in the cartContext, just like how shipping fee is.

Reply:
@akosasante sure, I put a random number to see if it was working. 
![image](https://github.com/NatachaKey/reactEcommerce/assets/103755090/54183ad2-95bc-4f51-8b82-5be6312cac28)
is that ok to write it like this? or is there a better approach?